## Finding Nature Specific Words

This notebook is designed to expand words under the `Nature` trope. Through research, the ASTRSC team has discovered that [this website](https://describeweb.com/describing-words-describe-nature/) contains most of the words ASTRSC should flag. However, this website contains over 600 words not all of which are relevant in the ASTRSC context. Hence, we will scrape this website and only choose the relevant words.

The script below follows standard scraping practices. Use Google or YouTube for more information. 

Note: because scraping is an iterative process that highly depends on how the website is designed, there isn't one manual to guide the user through scraping this particular website. Instead, the user could iteratively index into the website until the list containing the words is reached.

In [1]:
# necessary packages

import bs4

from urllib.request import urlopen as uReq

from bs4 import BeautifulSoup as bs

import requests

### Scraping The Website

These words are sourced [from this website](http://www.rhymezone.com/r/rhyme.cgi?Word=wildlife&org1=syl&org2=l&typeofrhyme=rel&org3=y).

In [4]:
# save url
my_url = "https://a-z-animals.com/animals/location/africa/algeria/"

# send request
uClient = uReq(my_url)

# read output
page_html = uClient.read()

# close connection
uClient.close()

page_soup = bs(page_html, "html.parser")

HTTPError: HTTP Error 403: Forbidden

In [3]:
# load url
r = requests.get(my_url)

# convert to a beautiful soup object
soup = bs(r.content)

Use the `prettify` function to print the output in a properly indexed format to easily identify how the words are encoded on the website.

In [4]:
print(soup.prettify())

<!DOCTYPE html>
<html lang="en">
 <head>
  <script>
   var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
    (function() {
        var gads = document.createElement('script');
        gads.async = true;
        gads.type = 'text/javascript';
        var useSSL = 'https:' == document.location.protocol;
        gads.src = (useSSL ? 'https:' : 'http:') + '//www.googletagservices.com/tag/js/gpt.js';
        var node = document.getElementsByTagName('script')[0];
        node.parentNode.insertBefore(gads, node);
    })();
  </script>
  <script>
   var gptAdSlots = [];
var sizeMappings = [];
googletag.cmd.push(function() {
    sizeMappings['RZ_Desktop_Left'] = googletag.sizeMapping()
      .addSize([1024, 200],[160, 600]) //Desktop
      .addSize([768, 200], [160, 600]) //Tablet
      .addSize([320, 100], [160, 600]) //Mobile
      .build();
    gptAdSlots[0] = googletag.defineSlot('/7765068/rz/RZ_Desktop_Left', [160, 600], 'RZ_Desktop_Left')
            .defineSizeMap

In [8]:
soup.find_all("script", attrs={"script": "javascript"})

[]

In [9]:
body = soup.body
print(body.prettify())

<body bgcolor="#e7f3ff" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
 <center>
  <table>
   <tr>
    <td valign="top" width="170">
     <div id="RZ_Desktop_Left" style="">
      <script>
       googletag.cmd.push(function() { googletag.display('RZ_Desktop_Left');});
      </script>
     </div>
    </td>
    <td>
    </td>
    <td valign="top">
     <center>
      <br/>
      <a href="/">
       <img border="0" src="/rz6.png"/>
      </a>
      <br/>
      <br/>
      <center>
       <div class="mainform">
        <form action="/r/rhyme.cgi" id="form1" method="GET" name="form1">
         <b>
          Word:
         </b>
         <input id="rzinput" maxlength="120" name="Word" size="15" style="font-size: 16px;" value="wildlife"/>
         <input style="font-size: 16px;" type="submit" value="Search"/>
         <select name="typeofrhyme" onchange="this.form.submit()" style="font-size: 16px;">
          <option value="perfect">
           Find rhymes
          </option>
 

In [10]:
body.find_all("script")

[<script>
     googletag.cmd.push(function() { googletag.display('RZ_Desktop_Left');});
   </script>,
 <script language="javascript" type="text/javascript">
 var THESAURUS_RESULTS =  [{"word":"wildlife","score":2147483647,"tags":["query","n"],"defs":["n\tall living things (except people) that are undomesticated"]},{"word":"fauna","score":72914,"tags":["n"],"defs":["n\tall the animal life in a particular region","n\ta living organism characterized by voluntary movement"]},{"word":"habitat","score":72405,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"]},{"word":"habitats","score":71610,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"],"defHeadword":"habitat"},{"word":"conservation","score":70874,"tags":["n"],"defs":["n\tthe preservation and careful management of the environment and of natural resources","n\t(physics) the maintenance of a certain quantities unchanged during che

In [11]:
target_script = body.find_all("script")[1]
print(target_script.prettify())

<script language="javascript" type="text/javascript">
 var THESAURUS_RESULTS =  [{"word":"wildlife","score":2147483647,"tags":["query","n"],"defs":["n\tall living things (except people) that are undomesticated"]},{"word":"fauna","score":72914,"tags":["n"],"defs":["n\tall the animal life in a particular region","n\ta living organism characterized by voluntary movement"]},{"word":"habitat","score":72405,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"]},{"word":"habitats","score":71610,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"],"defHeadword":"habitat"},{"word":"conservation","score":70874,"tags":["n"],"defs":["n\tthe preservation and careful management of the environment and of natural resources","n\t(physics) the maintenance of a certain quantities unchanged during chemical reactions or physical transformations","n\tan occurrence of improvement by virtue of preventing l

In [12]:
target_script.find_all("script", attrs={"type": "text/javascript"})


[]

In [13]:
target_script.get_text()

''

In [14]:
list_of_words = [{"word":"wildlife","score":2147483647,"tags":["query","n"],"defs":["n\tall living things (except people) that are undomesticated"]},{"word":"fauna","score":72914,"tags":["n"],"defs":["n\tall the animal life in a particular region","n\ta living organism characterized by voluntary movement"]},{"word":"habitat","score":72405,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"]},{"word":"habitats","score":71610,"tags":["n"],"defs":["n\tthe type of environment in which an organism or group normally lives or occurs"],"defHeadword":"habitat"},{"word":"conservation","score":70874,"tags":["n"],"defs":["n\tthe preservation and careful management of the environment and of natural resources","n\t(physics) the maintenance of a certain quantities unchanged during chemical reactions or physical transformations","n\tan occurrence of improvement by virtue of preventing loss or injury or other change"]},{"word":"animals","score":70786,"tags":["n"],"defs":["n\ta living organism characterized by voluntary movement"],"defHeadword":"animal"},{"word":"birds","score":69985,"tags":["n"],"defs":["n\twarm-blooded egg-laying vertebrates characterized by feathers and forelimbs modified as wings","n\tthe flesh of a bird or fowl (wild or domestic) used as food","n\tbadminton equipment consisting of a ball of cork or rubber with a crown of feathers","n\tinformal terms for a (young) woman","n\ta cry or noise made to express displeasure or contempt"],"defHeadword":"bird"},{"word":"animal","score":69555,"tags":["n","adj"],"defs":["n\ta living organism characterized by voluntary movement","adj\tof the nature of or characteristic of or derived from an animal or animals"]},{"word":"flora","score":69517,"tags":["n"],"defs":["n\ta living organism lacking the power of locomotion","n\tall the plant life in a particular region"]},{"word":"species","score":69165,"tags":["n"],"defs":["n\ta specific kind of something","n\t(biology) taxonomic group whose members can interbreed"]},{"word":"livestock","score":66938,"tags":["n"],"defs":["n\tnot used technically; any animals kept for use or profit"]},{"word":"hunting","score":65845,"tags":["n","v"],"defs":["n\tthe pursuit and killing or capture of wild animals regarded as a sport","n\tthe work of finding and killing or capturing animals for food or pelts","n\tthe activity of looking thoroughly in order to find something or someone"]},{"word":"parks","score":65753,"tags":["n"],"defs":["n\tUnited States civil rights leader who refused to give up her seat on a bus to a white man in Montgomery (Alabama) and so triggered the national civil rights movement (born in 1913)"]},{"word":"biota","score":65730,"tags":["n"],"defs":["n\tall the plant and animal life of a particular region"]},{"word":"geese","score":65357,"tags":["n"],"defs":["n\tweb-footed long-necked typically gregarious migratory aquatic birds usually larger and less aquatic than ducks","n\tflesh of a goose (domestic or wild)","n\ta man who is a stupid incompetent fool"],"defHeadword":"goose"},{"word":"wilderness","score":65249,"tags":["n"],"defs":["n\ta wild and uninhabited area"]},{"word":"wild","score":64062,"tags":["adj"],"defs":["adj\tin a state of extreme emotion","adj\tproduced without being planted or without human labor","adj\tin a natural state; not tamed or domesticated or cultivated","adj\tmarked by extreme lack of restraint or control","adj\tnot subjected to control or restraint","adj\tdeviating widely from an intended course","adj\t(of colors or sounds) intensely vivid or loud","adj\ttalking or behaving irrationally","adj\tlocated in a dismal or remote area; desolate","adj\t(of the elements) as if showing violent anger","adj\twithout civilizing influences"]},{"word":"park","score":63000,"tags":["n"],"defs":["n\ta gear position that acts as a parking brake","n\ta large area of land preserved in its natural state as public property","n\ta piece of open land for recreational use in an urban area","n\tScottish explorer in Africa (1771-1806)","n\ta facility in which ball games are played (especially baseball games)","n\ta lot where cars are parked"]},{"word":"natural","score":62136,"tags":["adj"],"defs":["adj\tbeing talented through inherited qualities","adj\trelated by blood; not adopted","adj\tin accordance with nature; relating to or concerning nature","adj\texisting in or produced by nature; not artificial or imitation","adj\texisting in or in conformity with nature or the observable world; neither supernatural nor magical","adj\tof a key containing no sharps or flats","adj\tfunctioning or occurring in a normal way; lacking abnormalities or deficiencies","adj\t(used especially of commodities) in the natural unprocessed condition","adj\tunthinking; prompted by (or as if by) instinct","adj\tunaffected and natural looking"]},{"word":"nature","score":61560,"tags":["n"],"defs":["n\tthe complex of emotional and intellectual attributes that determine a person's characteristic actions and reactions","n\tthe essential qualities or characteristics by which something is recognized","n\ta particular type of thing","n\tthe natural physical world including plants and animals and landscapes etc.","n\ta causal agent creating and controlling things in the universe"]},{"word":"creek","score":60901,"tags":["n"],"defs":["n\tany member of the Creek Confederacy (especially the Muskogee) formerly living in Georgia and Alabama but now chiefly in Oklahoma","n\ta natural stream of water smaller than a river (and often a tributary of a river)"]},{"word":"land","score":60777,"tags":["n"],"defs":["n\tthe solid part of the earth's surface","n\tmaterial in the top layer of the surface of the earth in which plants can grow (especially with reference to its quality or use)","n\tUnited States inventor who incorporated Polaroid film into lenses and invented the one-step photographic process (1909-1991)","n\tthe land on which real estate is located","n\tworking the land as an occupation or way of life","n\tthe people who live in a nation or country","n\textensive landed property (especially in the country) retained by the owner for his own use","n\ta domain in which something is dominant","n\tthe territory occupied by a nation","n\tterritory over which rule or control is exercised","n\ta politically organized body of people under a single government"]},{"word":"hunt","score":60378,"tags":["n","v"],"defs":["n\tthe pursuit and killing or capture of wild animals regarded as a sport","n\tthe work of finding and killing or capturing animals for food or pelts","n\tan instance of searching for something","n\tan association of huntsmen who hunt for sport","n\tBritish writer who defended the romanticism of Keats and Shelley (1784-1859)","n\tUnited States architect (1827-1895)","n\tEnglishman and Pre-Raphaelite painter (1827-1910)","n\tthe activity of looking thoroughly in order to find something or someone","v\tchase away, with as with force","v\tpursue for food or sport (as of wild animals)","v\tsearch (an area) for prey","v\tseek, search for","v\toscillate about a desired speed, position, or state to an undesirable extent","v\tyaw back and forth about a flight path","v\tpursue or chase relentlessly"]},{"word":"camping","score":60200,"tags":["n"],"defs":["n\tthe act of encamping and living in tents in a camp"]},{"word":"wildcat","score":58267,"tags":["n"],"defs":["n\tany small or medium-sized cat resembling the domestic cat and living in the wild","n\tan exploratory oil well drilled in land not known to be an oil field","n\ta cruelly rapacious person"]},{"word":"sauvage","score":56472,"tags":["n"]},{"word":"wwf","score":55474,"tags":["n","prop"]},{"word":"savage","score":55302,"tags":["n"],"defs":["n\ta member of an uncivilized people","n\ta cruelly rapacious person"]},{"word":"faun","score":55287,"tags":["n"],"defs":["n\tancient Italian deity in human shape, with horns, pointed ears and a goat's tail; equivalent to Greek satyr"]},{"word":"game","score":53947,"tags":["n"],"defs":["n\ta contest with rules to determine a winner","n\ta single play of a game","n\tan amusement or pastime","n\tfrivolous or trifling behavior","n\tyour occupation or line of work","n\tanimal hunted for food or sport","n\tthe game equipment needed to play a game","n\tthe flesh of wild animals that is used for food","n\t(games) the score at a particular point or the score needed to win","n\ta secret scheme to do something (especially something underhand or illegal)"]},{"word":"ground","score":53719,"tags":["n"],"defs":["n\t(art) the surface (as a wall or canvas) prepared to take the paint for a painting","n\ta connection between an electrical device and the earth (which is a zero voltage)","n\ta relatively homogeneous percept extending back of the figure on which attention is focused","n\ta position to be won or defended in battle (or as if in battle)","n\tthe first or preliminary coat of paint or size applied to a surface","n\tthe part of a scene (or picture) that lies behind objects in the foreground","n\ta rational motive for a belief or action","n\tmaterial in the top layer of the surface of the earth in which plants can grow (especially with reference to its quality or use)","n\tthe loose soft material that makes up a large part of the land surface","n\ta relation that provides the foundation for something","n\tthe solid part of the earth's surface"]},{"word":"faune","score":52373,"tags":["n","prop"]},{"word":"heathen","score":52144,"tags":["n"],"defs":["n\ta person who does not acknowledge your God"]},{"word":"ruthless","score":51988,"tags":["adj"],"defs":["adj\twithout mercy or pity"]},{"word":"cws","score":51024,"tags":["n"]},{"word":"land-based","score":51024,"tags":["n"]},{"word":"landmines","score":51024,"tags":["n"]},{"word":"mwa","score":51024,"tags":["n","prop"]},{"word":"natuur","score":51024,"tags":["n"]},{"word":"watchable","score":51024,"tags":["adj"]},{"word":"wild-type","score":51024,"tags":["n"]},{"word":"waterfowl","score":51020,"tags":["n","adj"],"defs":["n\tfreshwater aquatic bird"]},{"word":"deer","score":51019,"tags":["n"],"defs":["n\tdistinguished from Bovidae by the male's having solid deciduous antlers"]},{"word":"bighorn sheep","score":51018,"tags":["n"],"defs":["n\twild sheep of mountainous regions of western North America having massive curled horns"]},{"word":"ecology","score":51017,"tags":["n"],"defs":["n\tthe branch of biology concerned with the relations between organisms and their environment","n\tthe environment as it relates to living organisms"]},{"word":"biodiversity","score":51016,"tags":["n"],"defs":["n\tthe diversity of plant and animal life in a particular habitat (or in the world as a whole)"]},{"word":"forest","score":51015,"tags":["n","adj"],"defs":["n\tthe trees and other plants in a large densely wooded area","n\tland that is covered with trees and shrubs"]},{"word":"conservancy","score":51014,"tags":["n"],"defs":["n\tthe official conservation of trees and soil and rivers etc.","n\ta commission with jurisdiction over fisheries and navigation in a port or river"]},{"word":"bald eagle","score":51013,"tags":["n","prop"],"defs":["n\tlarge North American eagle having a white head and dark wings and body"]},{"word":"wildfowl","score":51012,"tags":["n"],"defs":["n\tflesh of any of a number of wild game birds suitable for food"]},{"word":"endangered species","score":51011,"tags":["n"],"defs":["n\ta species whose numbers are so small that the species is at risk of extinction"]},{"word":"conservationists","score":51010,"tags":["n"],"defs":["n\tsomeone who works to protect the environment from destruction or pollution"],"defHeadword":"conservationist"},{"word":"raptors","score":51009,"tags":["n"],"defs":["n\tany of numerous carnivorous birds that hunt and kill other animals"],"defHeadword":"raptor"},{"word":"birdlife","score":51007,"tags":["n"]},{"word":"wetlands","score":51006,"tags":["n"],"defs":["n\ta low area where the land is saturated with water"],"defHeadword":"wetland"},{"word":"otters","score":51005,"tags":["n"],"defs":["n\tfreshwater carnivorous mammal having webbed and clawed feet and dark brown fur","n\tthe fur of an otter"],"defHeadword":"otter"},{"word":"elk","score":51004,"tags":["n"],"defs":["n\tlarge northern deer with enormous flattened antlers in the male; called elk in Europe and moose in North America"]},{"word":"fisheries","score":51002,"tags":["n"],"defs":["n\ta workplace where fish are caught and processed and sold"],"defHeadword":"fishery"},{"word":"whitetail deer","score":51000,"tags":["n"],"defs":["n\tcommon North American deer; tail has a white underside"]},{"word":"bighorn","score":50999,"tags":["n"],"defs":["n\twild sheep of mountainous regions of western North America having massive curled horns","n\ta river that flows from central Wyoming to the Yellowstone River in southern Montana"]},{"word":"mule deer","score":50997,"tags":["n"],"defs":["n\tlong-eared deer of western North America with two-pronged antlers"]},{"word":"capercaillie","score":50996,"tags":["n"],"defs":["n\tlarge black Old World grouse"]},{"word":"whooping crane","score":50995,"tags":["n"],"defs":["n\trare North American crane having black-and-white plumage and a trumpeting call"]},{"word":"woodlands","score":50994,"tags":["n"],"defs":["n\tland that is covered with trees and shrubs"],"defHeadword":"woodland"},{"word":"avifauna","score":50993,"tags":["n"],"defs":["n\tthe birds of a particular region or period"]},{"word":"trumpeter swan","score":50992,"tags":["n"],"defs":["n\tlarge pure white wild swan of western North America having a sonorous cry"]},{"word":"shorebird","score":50991,"tags":["n"],"defs":["n\tany of numerous wading birds that frequent mostly seashores and estuaries"]},{"word":"waterbirds","score":50990,"tags":["n"],"defs":["n\tfreshwater aquatic bird"],"defHeadword":"waterbird"},{"word":"furbearer","score":50989,"tags":["n"]},{"word":"poachers","score":50988,"tags":["n"],"defs":["n\tsmall slender fish (to 8 inches) with body covered by bony plates; chiefly of deeper northern Pacific waters","n\ta cooking vessel designed to poach food (such as fish or eggs)","n\tsomeone who hunts or fishes illegally on the property of another"],"defHeadword":"poacher"},{"word":"massasauga","score":50987,"tags":["n"],"defs":["n\tpygmy rattlesnake found in moist areas from the Great Lakes to Mexico; feeds on mice and small amphibians","n\tsmall pygmy rattlesnake"]},{"word":"bird sanctuary","score":50986,"tags":["n"],"defs":["n\ta building where birds are kept"]},{"word":"lynx","score":50985,"tags":["n"],"defs":["n\tshort-tailed wildcats with usually tufted ears; valued for their fur"]},{"word":"ecosystems","score":50984,"tags":["n"],"defs":["n\ta system formed by the interaction of a community of organisms with their physical environment"],"defHeadword":"ecosystem"},{"word":"grizzlies","score":50983,"tags":["n"],"defs":["n\tpowerful brownish-yellow bear of the uplands of western North America"],"defHeadword":"grizzly"},{"word":"reptiles","score":50982,"tags":["n"],"defs":["n\tany cold-blooded vertebrate of the class Reptilia including tortoises turtles snakes lizards alligators crocodiles and extinct forms"],"defHeadword":"reptile"},{"word":"naturalists","score":50981,"tags":["n"],"defs":["n\ta biologist knowledgeable about natural history (especially botany and zoology)","n\tan advocate of the doctrine that the world can be understood in scientific terms"],"defHeadword":"naturalist"},{"word":"pronghorn","score":50980,"tags":["n"],"defs":["n\tfleet antelope-like ruminant of western North American plains with small branched horns"]},{"word":"prairie grouse","score":50979,"tags":["n"],"defs":["n\tbrown mottled North American grouse of western prairies"]},{"word":"coyotes","score":50978,"tags":["n"],"defs":["n\tsmall wolf native to western North America","n\ta forest fire fighter who is sent to battle remote and severe forest fires (often for days at a time)","n\tsomeone who smuggles illegal immigrants into the United States (usually acress the Mexican border)"],"defHeadword":"coyote"},{"word":"marine","score":50977,"tags":["n","adj"],"defs":["n\ta member of the United States Marine Corps","adj\tof or relating to the sea","adj\tof or relating to military personnel who serve both on land and at sea (specifically the U.S. Marine Corps)","adj\tnative to or inhabiting the sea","adj\trelating to or involving ships or shipping or navigation or seamen"]},{"word":"marsh harrier","score":50976,"tags":["n"],"defs":["n\tOld World harrier frequenting marshy regions"]},{"word":"bobolink","score":50975,"tags":["n"],"defs":["n\tmigratory American songbird"]},{"word":"critters","score":50974,"tags":["n"],"defs":["n\ta regional term for creature (especially domestic animals)"],"defHeadword":"critter"},{"word":"prairie dog","score":50973,"tags":["n"],"defs":["n\tany of several rodents of North American prairies living in large complex burrows having a barking cry"]},{"word":"bobwhite quail","score":50972,"tags":["n"],"defs":["n\ta popular North American game bird; named for its call"]},{"word":"waterways","score":50971,"tags":["n"],"defs":["n\ta navigable body of water","n\ta conduit through which water flows"],"defHeadword":"waterway"},{"word":"antelope","score":50970,"tags":["n"],"defs":["n\tgraceful Old World ruminant with long legs and horns directed upward and backward; includes gazelles; springboks; impalas; addax; gerenuks; blackbucks; dik-diks"]},{"word":"cougars","score":50969,"tags":["n"],"defs":["n\tlarge American feline resembling a lion"],"defHeadword":"cougar"},{"word":"river otter","score":50968,"tags":["n"],"defs":["n\tsociable aquatic animal widely distributed along streams and lake borders in North America"]},{"word":"moose","score":50967,"tags":["n"],"defs":["n\tlarge northern deer with enormous flattened antlers in the male; called elk in Europe and moose in North America"]},{"word":"herpetology","score":50966,"tags":["n"],"defs":["n\tthe branch of zoology concerned with reptiles and amphibians"]},{"word":"wildflower","score":50965,"tags":["n"],"defs":["n\twild or uncultivated flowering plant"]},{"word":"manatee","score":50964,"tags":["n"],"defs":["n\tsirenian mammal of tropical coastal waters of America; the flat tail is rounded"]},{"word":"ecotourism","score":50963,"tags":["n"],"defs":["n\ttourism to exotic or threatened ecosystems to observe wildlife or to help preserve nature"]},{"word":"seabird","score":50962,"tags":["n"],"defs":["n\ta bird that frequents coastal waters and the open ocean: gulls; pelicans; gannets; cormorants; albatrosses; petrels; etc."]},{"word":"turtles","score":50961,"tags":["n"],"defs":["n\tany of various aquatic and land reptiles having a bony shell and flipper-like limbs for swimming"],"defHeadword":"turtle"},{"word":"vegetation","score":50960,"tags":["n"],"defs":["n\tinactivity that is passive and monotonous, comparable to the inactivity of plant life","n\tan abnormal growth or excrescence (especially a warty excrescence on the valves of the heart)","n\tall the plant life in a particular region","n\tthe process of growth in plants"]},{"word":"tortoises","score":50959,"tags":["n"],"defs":["n\tusually herbivorous land turtles having clawed elephant-like limbs; worldwide in arid area except Australia and Antarctica"],"defHeadword":"tortoise"},{"word":"gnatcatchers","score":50958,"tags":["n"],"defs":["n\tvery small North American and South American warblers"],"defHeadword":"gnatcatcher"},{"word":"wood stork","score":50957,"tags":["n"],"defs":["n\tAmerican stork resembling the true ibises in having a downward-curved bill; inhabits wooded swamps of New World tropics","n\tOld World wood ibis"]},{"word":"amphibians","score":50956,"tags":["n"],"defs":["n\tcold-blooded vertebrate typically living on land but breeding in water; aquatic larvae undergo metamorphosis into adult form","n\tan airplane designed to take off and land on water","n\ta flat-bottomed motor vehicle that can travel on land or water"],"defHeadword":"amphibian"},{"word":"piping plover","score":50955,"tags":["n"],"defs":["n\tsmall plover of eastern North America"]},{"word":"wolves","score":50954,"tags":["n"],"defs":["n\tany of various predatory carnivorous canine mammals of North America and Eurasia that usually hunt in packs","n\ta man who is aggressive in making amorous advances to women","n\tGerman classical scholar who claimed that the Iliad and Odyssey were composed by several authors (1759-1824)","n\tAustrian composer (1860-1903)","n\ta cruelly rapacious person"],"defHeadword":"wolf"},{"word":"stone curlew","score":50953,"tags":["n"],"defs":["n\tlarge-headed large-eyed crepuscular or nocturnal shorebird of the Old World and tropical America having a thickened knee joint"]},{"word":"quoll","score":50952,"tags":["n"]},{"word":"prairie chicken","score":50951,"tags":["n"],"defs":["n\tbrown mottled North American grouse of western prairies"]},{"word":"foxes","score":50950,"tags":["n"],"defs":["n\talert carnivorous mammal with pointed muzzle and ears and a bushy tail; most are predators that do not hunt in packs","n\tthe Algonquian language of the Fox people","n\ta member of an Algonquian people formerly living west of Lake Michigan along the Fox River","n\tEnglish religious leader who founded the Society of Friends (1624-1691)","n\tEnglish statesman who supported American independence and the French Revolution (1749-1806)","n\tthe gray or reddish-brown fur of a fox","n\ta shifty deceptive person"],"defHeadword":"fox"},{"word":"timber rattlesnake","score":50949,"tags":["n"],"defs":["n\twidely distributed in rugged ground of eastern United States"]},{"word":"ornithologist","score":50948,"tags":["n"],"defs":["n\ta zoologist who studies birds"]},{"word":"sacred ibis","score":50947,"tags":["n"],"defs":["n\tAfrican ibis venerated by ancient Egyptians"]},{"word":"tigers","score":50946,"tags":["n"],"defs":["n\ta terrorist organization in Sri Lanka that began in 1970 as a student protest over the limited university access for Tamil students; currently seeks to establish an independent Tamil state called Eelam; relies on guerilla strategy including terrorist tactics that target key government and military personnel"]},{"word":"houbara","score":50945,"tags":["n"]},{"word":"plover","score":50944,"tags":["n"],"defs":["n\tany of numerous chiefly shorebirds of relatively compact build having straight bills and large pointed wings; closely related to the sandpipers"]},{"word":"prairie rattlesnake","score":50943,"tags":["n"],"defs":["n\twidely distributed between the Mississippi and the Rockies"]},{"word":"grizzly bear","score":50942,"tags":["n"],"defs":["n\tpowerful brownish-yellow bear of the uplands of western North America"]},{"word":"monarch butterfly","score":50941,"tags":["n"],"defs":["n\tlarge migratory American butterfly having deep orange wings with black and white markings; the larvae feed on milkweed"]},{"word":"sandhill crane","score":50940,"tags":["n"]},{"word":"desert tortoise","score":50939,"tags":["n"],"defs":["n\tburrowing tortoise of the arid western United States and northern Mexico; may be reclassified as a member of genus Xerobates"]},{"word":"goshawk","score":50938,"tags":["n"],"defs":["n\tlarge hawk of Eurasia and North America used in falconry"]},{"word":"weka","score":50937,"tags":["n"],"defs":["n\tflightless New Zealand rail of thievish disposition having short wings each with a spur used in fighting"]},{"word":"cirl bunting","score":50936,"tags":["n"]},{"word":"ptarmigans","score":50935,"tags":["n"],"defs":["n\tlarge arctic and subarctic grouse with feathered feet and usually white winter plumage"],"defHeadword":"ptarmigan"},{"word":"woodland caribou","score":50934,"tags":["n"],"defs":["n\tany of several large caribou living in coniferous forests of southern Canada; in some classifications included in the species Rangifer tarandus"]},{"word":"dotterel","score":50933,"tags":["n"],"defs":["n\trare plover of upland areas of Eurasia"]},{"word":"tahrs","score":50932,"tags":["n"]},{"word":"hellbender","score":50931,"tags":["n"],"defs":["n\tlarge salamander of North American rivers and streams"]},{"word":"cormorants","score":50929,"tags":["n"],"defs":["n\tlarge voracious dark-colored long-necked seabird with a distensible pouch for holding fish; used in Asia to catch fish"],"defHeadword":"cormorant"},{"word":"leopards","score":50928,"tags":["n"],"defs":["n\tlarge feline of African and Asian forests usually having a tawny coat with black spots","n\tthe pelt of a leopard"],"defHeadword":"leopard"},{"word":"pine marten","score":50927,"tags":["n"],"defs":["n\tdark brown marten of northern Eurasian coniferous forests"]},{"word":"panther","score":50926,"tags":["n"],"defs":["n\ta leopard in the black color phase","n\ta large spotted feline of tropical America similar to the leopard; in some classifications considered a member of the genus Felis","n\tlarge American feline resembling a lion"]},{"word":"ursus arctos horribilis","score":50925,"tags":["n","prop"],"defs":["n\tpowerful brownish-yellow bear of the uplands of western North America"]},{"word":"bobcat","score":50924,"tags":["n"],"defs":["n\tsmall lynx of North America"]},{"word":"sage grouse","score":50923,"tags":["n"],"defs":["n\tlarge grouse of sagebrush regions of North America"]},{"word":"alligators","score":50922,"tags":["n"],"defs":["n\teither of two amphibious reptiles related to crocodiles but with with shorter broader snouts","n\tleather made from alligator's hide"],"defHeadword":"alligator"},{"word":"xerosere","score":50921,"tags":["n"]},{"word":"zootechnics","score":50920,"tags":["n"]},{"word":"canadian goose","score":50919,"tags":["n","prop"],"defs":["n\tcommon grayish-brown wild goose of North America"]},{"word":"amphibiology","score":50917,"tags":["n"]},{"word":"oilbird","score":50916,"tags":["n"],"defs":["n\tnocturnal fruit-eating bird of South America that has fatty young yielding an oil that is used instead of butter"]},{"word":"safari park","score":50915,"tags":["n"],"defs":["n\tan area of parkland where wild animals are kept and can be viewed by visitors driving through"]},{"word":"seafowl","score":50914,"tags":["n"],"defs":["n\ta bird that frequents coastal waters and the open ocean: gulls; pelicans; gannets; cormorants; albatrosses; petrels; etc."]},{"word":"bird of passage","score":50913,"tags":["n"],"defs":["n\tany bird that migrates seasonally","n\tsomeone who leads a wandering unsettled life"]},{"word":"zoography","score":50912,"tags":["n"]},{"word":"zoological garden","score":50911,"tags":["n","prop"],"defs":["n\tthe facility where wild animals are housed for exhibition"]},{"word":"shore bird","score":50910,"tags":["n"],"defs":["n\tany of numerous wading birds that frequent mostly seashores and estuaries"]},{"word":"plain wanderer","score":50909,"tags":["n"],"defs":["n\tsmall Australian bird related to the button quail; classified as wading bird but inhabits plains"]},{"word":"zoo","score":50908,"tags":["n"],"defs":["n\ta facility where wild animals are housed for exhibition"]},{"word":"deer mouse","score":50907,"tags":["n"],"defs":["n\tbrownish New World mouse; most widely distributed member of the genus"]},{"word":"mammal","score":50906,"tags":["n"],"defs":["n\tany warm-blooded vertebrate having the skin more or less covered with hair; young are born alive except for the small subclass of monotremes and nourished with milk"]},{"word":"dinotherium","score":50905,"tags":["n"]},{"word":"bird","score":50904,"tags":["n"],"defs":["n\twarm-blooded egg-laying vertebrates characterized by feathers and forelimbs modified as wings","n\tthe flesh of a bird or fowl (wild or domestic) used as food","n\tbadminton equipment consisting of a ball of cork or rubber with a crown of feathers","n\tinformal terms for a (young) woman","n\ta cry or noise made to express displeasure or contempt"]},{"word":"conservationist","score":50903,"tags":["n"],"defs":["n\tsomeone who works to protect the environment from destruction or pollution"]},{"word":"gallery forest","score":50901,"tags":["n"]},{"word":"zoographer","score":50900,"tags":["n"]},{"word":"seriema","score":50899,"tags":["n"],"defs":["n\tBrazilian cariama; sole representative of the genus Cariama","n\tArgentinian cariama"]},{"word":"wetland","score":50898,"tags":["n","adj"],"defs":["n\ta low area where the land is saturated with water"]},{"word":"zoophagy","score":50897,"tags":["n"]},{"word":"predatory animal","score":50896,"tags":["n"],"defs":["n\tany animal that lives by preying on other animals"]},{"word":"bird table","score":50895,"tags":["n"]},{"word":"gayal","score":50894,"tags":["n"],"defs":["n\tox of southeast Asia sometimes considered a domesticated breed of the gaur"]},{"word":"ferae naturae","score":50893,"tags":["n"]},{"word":"nemophilist","score":50892,"tags":["n"]},{"word":"sea cow","score":50891,"tags":["n"],"defs":["n\tany of two families of large herbivorous aquatic mammals with paddle-shaped tails and flipper-like forelimbs and no hind limbs"]},{"word":"anhima","score":50890,"tags":["n"],"defs":["n\ttype genus of the Anhimidae; horned screamers"]},{"word":"titanothere","score":50889,"tags":["n"]},{"word":"birdlet","score":50888,"tags":["n"]},{"word":"taxonomic group","score":50887,"tags":["n"],"defs":["n\tanimal or plant group having natural relations"]},{"word":"tropical rain forest","score":50886,"tags":["n","prop"],"defs":["n\ta rain forest in a tropical area"]},{"word":"sea elephant","score":50885,"tags":["n"],"defs":["n\teither of two large northern Atlantic earless seals having snouts like trunks"]},{"word":"zootrophic","score":50884,"tags":["n"]},{"word":"ortygan","score":50883,"tags":["n"],"defs":["n\tany of several East Indian birds"]},{"word":"sea leopard","score":50882,"tags":["n"]},{"word":"guacharo","score":50881,"tags":["n"],"defs":["n\tnocturnal fruit-eating bird of South America that has fatty young yielding an oil that is used instead of butter"]},{"word":"gerfalcon","score":50880,"tags":["n"],"defs":["n\tlarge and rare arctic falcon having white and dark color phases"]},{"word":"animalness","score":50879,"tags":["n"]},{"word":"panzootic","score":50878,"tags":["n"]},{"word":"staffage","score":50877,"tags":["n"]},{"word":"water rail","score":50876,"tags":["n"]},{"word":"aquatic bird","score":50875,"tags":["n"],"defs":["n\twading and swimming and diving birds of either fresh or salt water"]},{"word":"everglades","score":50874,"tags":["n"],"defs":["n\ta large subtropical swamp in southern Florida that is noted for its wildlife"]},{"word":"birdwatch","score":50873,"tags":["n"],"defs":["v\twatch and study birds in their natural habitat"]},{"word":"zoopathology","score":50872,"tags":["n"]},{"word":"wild dog","score":50871,"tags":["n"],"defs":["n\tany of various undomesticated mammals of the family Canidae that are thought to resemble domestic dogs as distinguished from jackals or wolves"]},{"word":"dikkop","score":50870,"tags":["n"]},{"word":"marine animal","score":50869,"tags":["n"],"defs":["n\tany of numerous animals inhabiting the sea including e.g. fishes and molluscs and many mammals"]},{"word":"lernean","score":50868,"tags":["n","prop"]},{"word":"deerfly","score":50867,"tags":["n"]},{"word":"theriomorphism","score":50866,"tags":["n"]},{"word":"african elephant","score":50865,"tags":["n","prop"],"defs":["n\tAfrican elephant having enormous flapping ears and ivory tusks"]},{"word":"sanctuary","score":50864,"tags":["n"],"defs":["n\ta consecrated place where sacred objects are kept","n\tarea around the altar of a church for the clergy and choir; often enclosed by a lattice or railing","n\ta shelter from danger or hardship"]},{"word":"indian elephant","score":50863,"tags":["n","prop"],"defs":["n\tAsian elephant having smaller ears and tusks primarily in the male"]},{"word":"perissodactyl","score":50862,"tags":["n"],"defs":["n\tplacental mammals having hooves with an odd number of toes on each foot"]},{"word":"woodland","score":50861,"tags":["n"],"defs":["n\tland that is covered with trees and shrubs"]},{"word":"paleornithology","score":50860,"tags":["n"],"defs":["n\tthe paleobiology of birds"]},{"word":"viverrine","score":50859,"tags":["n"],"defs":["n\tsmall catlike predatory mammals of warmer parts of the Old World"]},{"word":"biocenosis","score":50858,"tags":["n"]},{"word":"elephant bird","score":50857,"tags":["n"],"defs":["n\thuge (to 9 ft.) extinct flightless bird of Madagascar"]},{"word":"guadalupe fur seal","score":50856,"tags":["n","prop"],"defs":["n\tof Pacific coast of California and southward"]},{"word":"birdling","score":50855,"tags":["n"]},{"word":"quadratojugal","score":50854,"tags":["n"]},{"word":"entoproct","score":50853,"tags":["n"],"defs":["n\tany of various mosslike aquatic animals usually forming branching colonies; each polyp having a both mouth and anus within a closed ring of tentacles"]},{"word":"black grouse","score":50852,"tags":["n","prop"],"defs":["n\tgrouse of which the male is bluish-black"]},{"word":"birdcatching","score":50851,"tags":["n"]},{"word":"prairie wolf","score":50850,"tags":["n"],"defs":["n\tsmall wolf native to western North America"]},{"word":"rangership","score":50849,"tags":["n"]},{"word":"linsang","score":50848,"tags":["n"]},{"word":"thremmatology","score":50847,"tags":["n"]},{"word":"sungrebe","score":50846,"tags":["n"]},{"word":"willow grouse","score":50845,"tags":["n"]},{"word":"snail darter","score":50844,"tags":["n"],"defs":["n\t3-inch snail-eating perch of the Tennessee River"]},{"word":"hairbird","score":50843,"tags":["n"]},{"word":"eskimo curlew","score":50842,"tags":["n","prop"],"defs":["n\tNew World curlew that breeds in northern North America"]},{"word":"snowshoe hare","score":50841,"tags":["n"],"defs":["n\tlarge large-footed North American hare; white in winter"]},{"word":"zoological","score":50840,"tags":["n","prop"],"defs":["adj\tof or relating to animals or animal groups","adj\tconcerning the study of animals and their classification and properties"]},{"word":"wildgrave","score":50839,"tags":["n"]},{"word":"ichthyoid","score":50838,"tags":["n"]},{"word":"frigate bird","score":50837,"tags":["n"],"defs":["n\tlong-billed warm-water seabird with wide wingspan and forked tail"]},{"word":"kangaroo rat","score":50836,"tags":["n"],"defs":["n\tany of various leaping rodents of desert regions of North America and Mexico; largest members of the family Heteromyidae","n\tany of several rabbit-sized rat-like Australian kangaroos"]},{"word":"procyonid","score":50835,"tags":["n"],"defs":["n\tplantigrade carnivorous mammals"]},{"word":"wrybill","score":50834,"tags":["n"]},{"word":"paradoxure","score":50833,"tags":["n"]},{"word":"paguma","score":50832,"tags":["n","prop"]},{"word":"much","score":41421,"tags":["adv"],"defs":["adv\t(degree adverb used before a noun phrase) for all practical purposes but not completely","adv\tvery","adv\tto a great degree or extent","adv\tfrequently or in great quantities","adv\tto a very great degree or extent"]},{"word":"alike","score":41420,"tags":["adv"],"defs":["adv\tin a like manner","adv\tequally"]},{"word":"other","score":41419,"tags":["adj"],"defs":["adj\tnot the same one or ones already mentioned or implied","adj\tvery unusual; different in character or quality from the normal or expected"]},{"word":"abundant","score":41418,"tags":["adj"],"defs":["adj\tpresent in great quantity"]},{"word":"local","score":41417,"tags":["adj"],"defs":["adj\taffecting only a restricted part or area of the body","adj\tof or belonging to or characteristic of a particular locality or neighborhood","adj\trelating to or applicable to or concerned with the administration of a city or town or district rather than a larger area"]},{"word":"native","score":41416,"tags":["adj"],"defs":["adj\tas found in nature in the elemental form","adj\tbelonging to one by birth","adj\tbeing such by origin","adj\tbeing or composed of people inhabiting a region from the beginning","adj\tnormally existing at birth"]},{"word":"african","score":41415,"tags":["adj","prop"],"defs":["adj\tof or relating to the nations of Africa or their peoples"]},{"word":"exotic","score":41413,"tags":["adj"],"defs":["adj\tstrikingly strange or unusual","adj\tbeing or from or characteristic of another place or part of the world"]},{"word":"indigenous","score":41412,"tags":["adj"],"defs":["adj\toriginating where it is found"]},{"word":"aquatic","score":41411,"tags":["adj"],"defs":["adj\trelating to or consisting of or being in water","adj\toperating or living or growing in water"]},{"word":"terrestrial","score":41410,"tags":["adj"],"defs":["adj\toperating or living or growing on land","adj\tconcerned with the world or worldly matters","adj\tof or relating to or characteristic of the planet Earth or its inhabitants","adj\tof or relating to or inhabiting the land as opposed to the sea or air","adj\tof this earth"]},{"word":"unique","score":41409,"tags":["adj"],"defs":["adj\thighly unusual or rare but not the single instance","adj\tthe single one of its kind","adj\tradically distinctive and without equal"]},{"word":"urban","score":41408,"tags":["adj"],"defs":["adj\tlocated in or characteristic of a city or city life","adj\trelating to or concerned with a city or densely populated area"]},{"word":"rich","score":41407,"tags":["adj"],"defs":["adj\tpleasantly full and mellow","adj\thighly seasoned or containing large amounts of choice ingredients such as butter or sugar or eggs","adj\tvery productive","adj\tpossessing material wealth","adj\thaving an abundant supply of desirable qualities or substances (especially natural resources)","adj\tsuggestive of or characterized by great expense","adj\thigh in mineral content; having a a high proportion of fuel to air","adj\tof great worth or quality","adj\tmarked by richness and fullness of flavor","adj\tstrong; intense","adj\tmarked by great fruitfulness","adj\taffording an abundant supply"]},{"word":"rare","score":41406,"tags":["adj"],"defs":["adj\tmarked by an uncommon quality; especially superlative or extreme of its kind","adj\thaving low density","adj\tnot widely distributed","adj\tnot widely known; especially valued for its uncommonness","adj\t(of meat) cooked a short time; still red inside","adj\trecurring only at long intervals"]},{"word":"diverse","score":41405,"tags":["adj"],"defs":["adj\tdistinctly dissimilar or unlike","adj\tmany and different"]},{"word":"australian","score":41404,"tags":["adj","prop"],"defs":["adj\tof or relating to or characteristic of Australia or its inhabitants"]},{"word":"endangered","score":41403,"tags":["adj","n","v"],"defs":["adj\t(of flora or fauna) in imminent danger of extinction"]},{"word":"migratory","score":41402,"tags":["adj"],"defs":["adj\tused of animals that move seasonally","adj\thabitually moving from place to place especially in search of seasonal work"]},{"word":"varied","score":41401,"tags":["adj"],"defs":["adj\tcharacterized by variety","adj\tbroken away from sameness or identity or duplication","adj\twidely different"]},{"word":"north","score":41400,"tags":["adj"],"defs":["adj\tsituated in or facing or moving toward or coming from the north"]},{"word":"protected","score":41399,"tags":["adj","v"],"defs":["adj\tkept safe or defended from danger or injury or loss","adj\tguarded from injury or destruction"]},{"word":"plentiful","score":41398,"tags":["adj"],"defs":["adj\texisting in great number or quantity","adj\tproducing in abundance","adj\taffording an abundant supply"]},{"word":"nongame","score":41397,"tags":["adj"]},{"word":"resident","score":41396,"tags":["adj"],"defs":["adj\tliving in a particular place","adj\tused of animals that do not migrate"]},{"word":"dangerous","score":41395,"tags":["adj"],"defs":["adj\tinvolving or causing danger or risk; liable to hurt or harm","adj\tcausing fear or anxiety by threatening great harm"]},{"word":"spectacular","score":41393,"tags":["adj"],"defs":["adj\tof the nature of spectacle or drama","adj\tsensational in appearance or thrilling in effect","adj\thaving a quality that thrusts itself into attention"]},{"word":"unusual","score":41392,"tags":["adj"],"defs":["adj\tnot commonly encountered","adj\tnot usual or common or ordinary","adj\tbeing definitely out of the ordinary and unexpected; slightly odd or even a bit weird"]},{"word":"tropical","score":41391,"tags":["adj"],"defs":["adj\tof weather or climate; hot and humid as in the tropics","adj\tcharacterized by or of the nature of a trope or tropes; changed from its literal sense","adj\trelating to or situated in or characteristic of the tropics (the region on either side of the equator)","adj\tof or relating to the tropics, or either tropic"]},{"word":"alaskan","score":41390,"tags":["adj"],"defs":["adj\trelating to or characteristic of the state or people of Alaska"]},{"word":"coastal","score":41389,"tags":["adj"],"defs":["adj\tof or relating to a coast","adj\tlocated on or near or bordering on a coast"]},{"word":"dependent","score":41388,"tags":["adj"],"defs":["adj\taddicted to a drug","adj\tcontingent on something else","adj\tnot independent","adj\tof a clause; unable to stand alone syntactically as a complete sentence","adj\tbeing under the power or sovereignty of another or others"]},{"word":"captive","score":41387,"tags":["adj"],"defs":["adj\tin captivity","adj\tdeeply moved"]},{"word":"injured","score":41386,"tags":["adj"],"defs":["adj\tusually used of physical or mental injury to persons"]},{"word":"recreational","score":41385,"tags":["adj"]},{"word":"vertebrate","score":41383,"tags":["adj"]},{"word":"arctic","score":41382,"tags":["adj","prop"]},{"word":"threatened","score":41381,"tags":["adj","v"]},{"word":"fascinating","score":41380,"tags":["adj"]},{"word":"nocturnal","score":41379,"tags":["adj"]},{"word":"avian","score":41378,"tags":["adj"]},{"word":"mammalian","score":41377,"tags":["adj"]},{"word":"protect","score":41376,"tags":["adj","v"]},{"word":"precious","score":41375,"tags":["adj"]},{"word":"associated","score":41374,"tags":["adj"]},{"word":"prolific","score":41373,"tags":["adj"]},{"word":"amazing","score":41372,"tags":["adj"]},{"word":"magnificent","score":41371,"tags":["adj"]},{"word":"antarctic","score":41370,"tags":["adj","prop"]},{"word":"desirable","score":41369,"tags":["adj"]},{"word":"respect","score":41368,"tags":["adj","v"]},{"word":"scenic","score":41367,"tags":["adj"]},{"word":"bountiful","score":41366,"tags":["adj"]},{"word":"assorted","score":41365,"tags":["adj"]},{"word":"beneficial","score":41364,"tags":["adj"]},{"word":"northeastern","score":41363,"tags":["adj","prop"]},{"word":"endemic","score":41362,"tags":["adj"]},{"word":"ocean","score":41361,"tags":["adj"]},{"word":"infected","score":41360,"tags":["adj"]},{"word":"underwater","score":41359,"tags":["adj"]},{"word":"swedish","score":41358,"tags":["adj","prop"]},{"word":"nontarget","score":41357,"tags":["adj"]},{"word":"winged","score":41356,"tags":["adj"]},{"word":"incredible","score":41355,"tags":["adj"]},{"word":"unwanted","score":41354,"tags":["adj"]},{"word":"elusive","score":41352,"tags":["adj"]},{"word":"amazonian","score":41351,"tags":["adj"]},{"word":"predatory","score":41350,"tags":["adj"]},{"word":"stuffed","score":41348,"tags":["adj"]},{"word":"orphaned","score":41346,"tags":["adj"]},{"word":"fish","score":41345,"tags":["n","v"]},{"word":"plants","score":41339,"tags":["n"]},{"word":"variety","score":41337,"tags":["n"]},{"word":"protection","score":41336,"tags":["n"]},{"word":"forests","score":41333,"tags":["n"]},{"word":"humans","score":41332,"tags":["n"]},{"word":"department","score":41331,"tags":["n"]},{"word":"environment","score":41330,"tags":["n"]},{"word":"resources","score":41329,"tags":["n"]},{"word":"management","score":41328,"tags":["n"]},{"word":"defenders","score":41327,"tags":["n"]},{"word":"forms","score":41325,"tags":["n"]},{"word":"division","score":41324,"tags":["n"]},{"word":"conference","score":41323,"tags":["n"]},{"word":"abundance","score":41322,"tags":["n"]},{"word":"scenery","score":41320,"tags":["n"]},{"word":"recreation","score":41319,"tags":["n"]},{"word":"impact","score":41318,"tags":["n"]},{"word":"preservation","score":41317,"tags":["n"]},{"word":"transactions","score":41316,"tags":["n"]},{"word":"food","score":41315,"tags":["n"]},{"word":"forestry","score":41313,"tags":["n"]},{"word":"kinds","score":41312,"tags":["n"]},{"word":"diversity","score":41311,"tags":["n"]},{"word":"tourism","score":41310,"tags":["n"]},{"word":"destruction","score":41309,"tags":["n"]},{"word":"landscape","score":41308,"tags":["n"]},{"word":"threat","score":41306,"tags":["n"]},{"word":"trees","score":41305,"tags":["n"]},{"word":"damage","score":41303,"tags":["n","v"]},{"word":"bureau","score":41302,"tags":["n"]},{"word":"soil","score":41301,"tags":["n"]},{"word":"beauty","score":41298,"tags":["n"]},{"word":"populations","score":41297,"tags":["n"]},{"word":"landscapes","score":41296,"tags":["n"]},{"word":"agriculture","score":41295,"tags":["n"]},{"word":"impacts","score":41294,"tags":["n"]},{"word":"timber","score":41291,"tags":["n"]},{"word":"haven","score":41290,"tags":["n"]},{"word":"refuge","score":41289,"tags":["n"]},{"word":"countryside","score":41288,"tags":["n"]},{"word":"plenty","score":41286,"tags":["n"]},{"word":"array","score":41285,"tags":["n"]},{"word":"geology","score":41282,"tags":["n"]},{"word":"lands","score":41281,"tags":["n"]},{"word":"lots","score":41280,"tags":["n"]},{"word":"uses","score":41279,"tags":["n"]},{"word":"cattle","score":41276,"tags":["n"]},{"word":"exploitation","score":41274,"tags":["n"]},{"word":"climate","score":41273,"tags":["n"]},{"word":"threats","score":41272,"tags":["n"]},{"word":"insects","score":41271,"tags":["n"]},{"word":"soils","score":41269,"tags":["n"]},{"word":"heritage","score":41268,"tags":["n"]},{"word":"disturbance","score":41267,"tags":["n"]},{"word":"killing","score":41266,"tags":["n","v"]},{"word":"wildlands","score":41265,"tags":["n"]},{"word":"crops","score":41263,"tags":["n"]},{"word":"wildflowers","score":41262,"tags":["n"]},{"word":"pesticides","score":41261,"tags":["n"]},{"word":"slaughter","score":41260,"tags":["n"]},{"word":"fishing","score":41259,"tags":["n"]},{"word":"utilization","score":41258,"tags":["n"]},{"word":"pets","score":41257,"tags":["n"]},{"word":"rabies","score":41256,"tags":["n"]},{"word":"bears","score":41255,"tags":["n"]},{"word":"beaches","score":41254,"tags":["n"]},{"word":"toxicity","score":41253,"tags":["n"]},{"word":"reserves","score":41252,"tags":["n"]},{"word":"pollution","score":41251,"tags":["n"]},{"word":"extinction","score":41250,"tags":["n"]},{"word":"depletion","score":41249,"tags":["n"]},{"word":"woods","score":41248,"tags":["n"]},{"word":"refuges","score":41247,"tags":["n","v"]},{"word":"disappearance","score":41246,"tags":["n"]},{"word":"encounters","score":41245,"tags":["n","v"]},{"word":"weather","score":41244,"tags":["n"]},{"word":"disruption","score":41243,"tags":["n"]},{"word":"photography","score":41242,"tags":["n"]},{"word":"utilisation","score":41241,"tags":["n"]},{"word":"herds","score":41240,"tags":["n"]},{"word":"sanctuaries","score":41239,"tags":["n"]},{"word":"ecosystem","score":41238,"tags":["n"]},{"word":"toll","score":41237,"tags":["n"]},{"word":"terrain","score":41236,"tags":["n"]},{"word":"havens","score":41235,"tags":["n"]},{"word":"elephants","score":41234,"tags":["n"]},{"word":"assortment","score":41233,"tags":["n"]},{"word":"contaminants","score":41232,"tags":["n"]},{"word":"profusion","score":41230,"tags":["n"]},{"word":"mammals","score":41229,"tags":["n"]},{"word":"taking","score":41228,"tags":["n"]},{"word":"outdoors","score":41227,"tags":["n"]},{"word":"ducks","score":41226,"tags":["n"]},{"word":"corridors","score":41225,"tags":["n"]},{"word":"displays","score":41224,"tags":["n"]},{"word":"poisoning","score":41223,"tags":["n","v"]},{"word":"harvest","score":41222,"tags":["n"]},{"word":"watch","score":41221,"tags":["n","v"]},{"word":"poaching","score":41220,"tags":["n","v"]},{"word":"richness","score":41219,"tags":["n"]},{"word":"harvesting","score":41218,"tags":["n","v"]},{"word":"lovers","score":41216,"tags":["n"]},{"word":"lujan","score":41215,"tags":["n","prop"]},{"word":"watersheds","score":41214,"tags":["n"]},{"word":"decimation","score":41213,"tags":["n"]},{"word":"harm","score":41211,"tags":["n","v"]},{"word":"alaska","score":41210,"tags":["n","prop"]},{"word":"glimpse","score":41209,"tags":["n"]},{"word":"trails","score":41208,"tags":["n"]},{"word":"rabbits","score":41207,"tags":["n"]},{"word":"waterfalls","score":41206,"tags":["n"]},{"word":"vines","score":41205,"tags":["n"]},{"word":"scenics","score":41204,"tags":["n"]},{"word":"forage","score":41203,"tags":["n"]},{"word":"glimpses","score":41202,"tags":["n"]},{"word":"photos","score":41201,"tags":["n"]},{"word":"wonders","score":41200,"tags":["n"]},{"word":"plight","score":41199,"tags":["n"]},{"word":"invertebrates","score":41198,"tags":["n"]},{"word":"feeding","score":41197,"tags":["n","v"]},{"word":"extermination","score":41196,"tags":["n"]},{"word":"amenity","score":41195,"tags":["n"]},{"word":"pastoralists","score":41194,"tags":["n"]},{"word":"shellfish","score":41193,"tags":["n"]},{"word":"livelihoods","score":41192,"tags":["n"]},{"word":"exhibits","score":41191,"tags":["n","v"]},{"word":"hazard","score":41190,"tags":["n"]},{"word":"harassment","score":41188,"tags":["n"]},{"word":"detriment","score":41187,"tags":["n"]},{"word":"whales","score":41186,"tags":["n"]},{"word":"marshes","score":41185,"tags":["n"]},{"word":"atlas","score":41183,"tags":["n"]},{"word":"buffalo","score":41181,"tags":["n"]},{"word":"glaciers","score":41180,"tags":["n"]},{"word":"sightings","score":41179,"tags":["n"]},{"word":"songbirds","score":41178,"tags":["n"]},{"word":"aesthetics","score":41177,"tags":["n"]},{"word":"eagles","score":41176,"tags":["n"]},{"word":"foliage","score":41175,"tags":["n"]},{"word":"commercialization","score":41174,"tags":["n"]},{"word":"nat","score":41173,"tags":["n"]},{"word":"seals","score":41172,"tags":["n"]},{"word":"importation","score":41171,"tags":["n"]},{"word":"squirrels","score":41170,"tags":["n"]},{"word":"tourists","score":41169,"tags":["n"]},{"word":"vistas","score":41168,"tags":["n"]},{"word":"nepal","score":41167,"tags":["n","prop"]},{"word":"preserve","score":41165,"tags":["v"]},{"word":"includes","score":41164,"tags":["v"]},{"word":"protecting","score":41163,"tags":["v"]},{"word":"viewing","score":41162,"tags":["v"]},{"word":"conserve","score":41160,"tags":["v"]},{"word":"teeming","score":41159,"tags":["v"]},{"word":"watching","score":41158,"tags":["v"]},{"word":"attract","score":41157,"tags":["v"]},{"word":"view","score":41155,"tags":["v"]},{"word":"observe","score":41154,"tags":["v"]},{"word":"seeing","score":41152,"tags":["v"]},{"word":"affect","score":41151,"tags":["v"]},{"word":"preserves","score":41150,"tags":["v"]},{"word":"conserving","score":41149,"tags":["v"]},{"word":"observing","score":41148,"tags":["v"]},{"word":"abounds","score":41147,"tags":["v"]},{"word":"managing","score":41146,"tags":["v"]},{"word":"save","score":41145,"tags":["v"]},{"word":"benefit","score":41144,"tags":["v"]},{"word":"preserving","score":41143,"tags":["v"]},{"word":"manage","score":41142,"tags":["v"]},{"word":"support","score":41140,"tags":["v"]},{"word":"kill","score":41139,"tags":["v"]},{"word":"affected","score":41137,"tags":["v"]},{"word":"disturb","score":41136,"tags":["v"]},{"word":"photographing","score":41135,"tags":["v"]},{"word":"enjoy","score":41134,"tags":["v"]},{"word":"saving","score":41133,"tags":["v"]},{"word":"grazing","score":41132,"tags":["v"]},{"word":"managed","score":41131,"tags":["v"]},{"word":"study","score":41130,"tags":["v"]},{"word":"spot","score":41129,"tags":["v"]},{"word":"destroyed","score":41128,"tags":["v"]},{"word":"studying","score":41124,"tags":["v"]},{"word":"vanishing","score":41123,"tags":["v"]},{"word":"feed","score":41122,"tags":["v"]},{"word":"spotting","score":41121,"tags":["v"]},{"word":"disturbing","score":41120,"tags":["v"]},{"word":"attracting","score":41119,"tags":["v"]},{"word":"affecting","score":41118,"tags":["v"]},{"word":"destroy","score":41117,"tags":["v"]},{"word":"threaten","score":41116,"tags":["v"]},{"word":"encourage","score":41114,"tags":["v"]},{"word":"owns","score":41113,"tags":["v"]},{"word":"encounter","score":41112,"tags":["v"]},{"word":"supports","score":41111,"tags":["v"]},{"word":"abounded","score":41110,"tags":["v"]},{"word":"hunted","score":41109,"tags":["v"]},{"word":"survive","score":41108,"tags":["v"]},{"word":"attracts","score":41107,"tags":["v"]},{"word":"destroying","score":41106,"tags":["v"]},{"word":"decimated","score":41104,"tags":["v"]},{"word":"eaten","score":41103,"tags":["v"]},{"word":"lives","score":41102,"tags":["v"]},{"word":"teemed","score":41101,"tags":["v"]},{"word":"teems","score":41100,"tags":["v"]},{"word":"inhabited","score":41099,"tags":["v"]},{"word":"protects","score":41098,"tags":["v"]},{"word":"disappearing","score":41097,"tags":["v"]},{"word":"endanger","score":41096,"tags":["v"]},{"word":"threatens","score":41094,"tags":["v"]},{"word":"sustain","score":41093,"tags":["v"]},{"word":"interested","score":41092,"tags":["v"]},{"word":"shooting","score":41091,"tags":["v"]},{"word":"harass","score":41090,"tags":["v"]},{"word":"kills","score":41089,"tags":["v"]},{"word":"thrives","score":41088,"tags":["v"]},{"word":"injures","score":41087,"tags":["v"]},{"word":"inhabits","score":41086,"tags":["v"]},{"word":"depleted","score":41085,"tags":["v"]},{"word":"compete","score":41084,"tags":["v"]},{"word":"scare","score":41083,"tags":["v"]},{"word":"inhabiting","score":41082,"tags":["v"]},{"word":"teem","score":41081,"tags":["v"]},{"word":"appreciate","score":41080,"tags":["v"]},{"word":"disturbed","score":41079,"tags":["v"]},{"word":"shoot","score":41078,"tags":["v"]},{"word":"threatening","score":41077,"tags":["v"]},{"word":"valuing","score":41076,"tags":["v"]},{"word":"abound","score":41075,"tags":["v"]},{"word":"restoring","score":41074,"tags":["v"]},{"word":"safeguard","score":41073,"tags":["v"]},{"word":"tracking","score":41072,"tags":["v"]},{"word":"harmed","score":41071,"tags":["v"]},{"word":"harming","score":41070,"tags":["v"]},{"word":"enjoying","score":41069,"tags":["v"]},{"word":"fascinated","score":41068,"tags":["v"]},{"word":"chase","score":41066,"tags":["v"]},{"word":"conserved","score":41065,"tags":["v"]},{"word":"dwindling","score":41064,"tags":["v"]},{"word":"caring","score":41063,"tags":["v"]},{"word":"monitor","score":41062,"tags":["v"]},{"word":"pose","score":41060,"tags":["v"]},{"word":"migrating","score":41059,"tags":["v"]},{"word":"disturbs","score":41058,"tags":["v"]},{"word":"suffers","score":41057,"tags":["v"]},{"word":"endangering","score":41056,"tags":["v"]},{"word":"filming","score":41055,"tags":["v"]},{"word":"depicting","score":41054,"tags":["v"]},{"word":"sustaining","score":41053,"tags":["v"]},{"word":"flourishes","score":41052,"tags":["v"]},{"word":"thrive","score":41051,"tags":["v"]},{"word":"populated","score":41050,"tags":["v"]},{"word":"displaced","score":41048,"tags":["v"]},{"word":"featuring","score":41047,"tags":["v"]},{"word":"inhabit","score":41045,"tags":["v"]},{"word":"features","score":41044,"tags":["v"]},{"word":"disrupt","score":41043,"tags":["v"]},{"word":"destroys","score":41042,"tags":["v"]},{"word":"diminishing","score":41041,"tags":["v"]},{"word":"painting","score":41040,"tags":["v"]},{"word":"survives","score":41039,"tags":["v"]},{"word":"abounding","score":41038,"tags":["v"]},{"word":"value","score":41036,"tags":["v"]},{"word":"capturing","score":41035,"tags":["v"]},{"word":"flourish","score":41034,"tags":["v"]},{"word":"landscaping","score":41032,"tags":["v"]},{"word":"paint","score":41031,"tags":["v"]},{"word":"trapping","score":41030,"tags":["v"]},{"word":"scaring","score":41029,"tags":["v"]},{"word":"harms","score":41028,"tags":["v"]},{"word":"exterminated","score":41027,"tags":["v"]},{"word":"brimming","score":41026,"tags":["v"]},{"word":"publishes","score":41025,"tags":["v"]},{"word":"warden","score":6424,"tags":["n"]},{"word":"attu","score":6364,"tags":["n","prop"]},{"word":"big bend national park","score":6364,"tags":["n"]},{"word":"billion-dollar grass","score":6364,"tags":["n"]},{"word":"bladder senna","score":6364,"tags":["n"]},{"word":"broads","score":6364,"tags":["n"]},{"word":"capitol reef national park","score":6364,"tags":["n","prop"]},{"word":"carson","score":6364,"tags":["n","prop"]},{"word":"cash","score":6364,"tags":["n"]},{"word":"cfs","score":6364,"tags":["n"]},{"word":"chronic wasting disease","score":6364,"tags":["n"]},{"word":"colutea arborescens","score":6364,"tags":["n"]},{"word":"covert","score":6364,"tags":["adj"]},{"word":"ddt","score":6364,"tags":["n"]},{"word":"diorama","score":6364,"tags":["n"]},{"word":"echinochloa frumentacea","score":6364,"tags":["n","prop"]},{"word":"forest ranger","score":6364},{"word":"fws","score":6364,"tags":["n"]},{"word":"game warden","score":6364,"tags":["n"]},{"word":"gamekeeper","score":6364,"tags":["n"]},{"word":"japanese barnyard millet","score":6364,"tags":["n","prop"]},{"word":"japanese millet","score":6364,"tags":["n","prop"]},{"word":"jasper national park","score":6364,"tags":["prop"]},{"word":"kruger national park","score":6364,"tags":["prop"]},{"word":"lair","score":6364,"tags":["n"]},{"word":"leakey","score":6364,"tags":["n","prop"]},{"word":"natural resources","score":6364,"tags":["n"]},{"word":"nest","score":6364,"tags":["n"]},{"word":"noah","score":6364,"tags":["n","prop"]},{"word":"norfolk","score":6364,"tags":["n","prop"]},{"word":"panda","score":6364,"tags":["n"]},{"word":"patagonia","score":6364,"tags":["n","prop"]},{"word":"pole","score":6364,"tags":["n"]},{"word":"poles","score":6364,"tags":["n"]},{"word":"pond","score":6364,"tags":["n"]},{"word":"preservationist","score":6364,"tags":["n"]},{"word":"preserved","score":6364,"tags":["adj"]},{"word":"rachel carson","score":6364,"tags":["n","prop"]},{"word":"rachel louise carson","score":6364,"tags":["n","prop"]},{"word":"reservation","score":6364,"tags":["n"]},{"word":"richard erskine leakey","score":6364,"tags":["n","prop"]},{"word":"richard leakey","score":6364,"tags":["n","prop"]},{"word":"safari","score":6364,"tags":["n"]},{"word":"sanwa millet","score":6364,"tags":["n","prop"]},{"word":"sar","score":6364,"tags":["n"]},{"word":"serengeti","score":6364,"tags":["n","prop"]},{"word":"spoor","score":6364,"tags":["n"]},{"word":"united states fish and wildlife service","score":6364,"tags":["n"]},{"word":"us fish and wildlife service","score":6364,"tags":["n"]},{"word":"agricultural","score":5304,"tags":["adj"]},{"word":"asylum","score":5060,"tags":["n"]},{"word":"biosphere","score":5060,"tags":["n"]},{"word":"cial","score":5060,"tags":["adj"]},{"word":"conifers","score":5060,"tags":["n"]},{"word":"correctional","score":5060,"tags":["adj"]},{"word":"curios","score":5060,"tags":["n"]},{"word":"environmental","score":5060,"tags":["adj"]},{"word":"freshwater","score":5060,"tags":["adj"]},{"word":"genera","score":5060,"tags":["n"]},{"word":"geriatric","score":5060,"tags":["adj"]},{"word":"herbivores","score":5060,"tags":["n"]},{"word":"inpatient","score":5060,"tags":["n"]},{"word":"manpower","score":5060,"tags":["n"]},{"word":"multilingual","score":5060,"tags":["adj","prop"]},{"word":"offsite","score":5060,"tags":["adj","n"]},{"word":"produce","score":5060,"tags":["v"]},{"word":"seabirds","score":5060,"tags":["n"]},{"word":"watershed","score":5060,"tags":["n"]},{"word":"wildland","score":5060,"tags":["adj","n"]}]


In [15]:
all_words = []

for i in range(len(list_of_words)):
    a_word = list_of_words[i]['word']
    all_words.append(a_word)

In [16]:
print(len(all_words))
print(all_words)

634
['wildlife', 'fauna', 'habitat', 'habitats', 'conservation', 'animals', 'birds', 'animal', 'flora', 'species', 'livestock', 'hunting', 'parks', 'biota', 'geese', 'wilderness', 'wild', 'park', 'natural', 'nature', 'creek', 'land', 'hunt', 'camping', 'wildcat', 'sauvage', 'wwf', 'savage', 'faun', 'game', 'ground', 'faune', 'heathen', 'ruthless', 'cws', 'land-based', 'landmines', 'mwa', 'natuur', 'watchable', 'wild-type', 'waterfowl', 'deer', 'bighorn sheep', 'ecology', 'biodiversity', 'forest', 'conservancy', 'bald eagle', 'wildfowl', 'endangered species', 'conservationists', 'raptors', 'birdlife', 'wetlands', 'otters', 'elk', 'fisheries', 'whitetail deer', 'bighorn', 'mule deer', 'capercaillie', 'whooping crane', 'woodlands', 'avifauna', 'trumpeter swan', 'shorebird', 'waterbirds', 'furbearer', 'poachers', 'massasauga', 'bird sanctuary', 'lynx', 'ecosystems', 'grizzlies', 'reptiles', 'naturalists', 'pronghorn', 'prairie grouse', 'coyotes', 'marine', 'marsh harrier', 'bobolink', 'cri

## Manually Pick Out Relevant Words

We will sift through the 634 words extracted from the website and filter those ASTRSC will flag.

In [20]:
selected_nature_words = ['wildlife', 'fauna', 'habitat', 'habitats', 'conservation', 'animals', 'birds', 'animal', 'flora', 'livestock', 'hunting', 'parks', 'biota', 'geese', 'wilderness', 'wild', 'park', 'natural', 'nature', 'hunt', 'camping', 'wildcat', 'sauvage', 'savage', 'faun', 'faune', 'wild-type', 'waterfowl', 'deer', 'bighorn sheep', 'ecology', 'biodiversity', 'forest', 'conservancy', 'bald eagle', 'wildfowl', 'endangered species', 'conservationists', 'raptors', 'birdlife', 'wetlands', 'otters', 'elk', 'fisheries', 'whitetail deer', 'bighorn', 'mule deer', 'capercaillie', 'waterbirds', 'poachers', 'bird sanctuary', 'ecosystems', 'grizzlies', 'reptiles', 'naturalists', 'coyotes', 'marine', 'waterways', 'antelope', 'cougars', 'river otter', 'moose', 'wildflower', 'ecotourism', 'seabird', 'turtles', 'vegetation', 'tortoises', 'gnatcatchers', 'wood stork', 'amphibians', 'wolves', 'stone curlew', 'foxes', 'timber rattlesnake', 'sacred ibis', 'tigers', 'grizzly bear', 'monarch butterfly', 'sandhill crane', 'desert tortoise', 'goshawk', 'cirl bunting', 'leopards', 'panther', 'bobcat', 'alligators', 'oilbird', 'safari park', 'seafowl', 'bird of passage', 'zoography', 'zoological garden', 'shore bird', 'plain wanderer', 'zoo', 'deer mouse', 'mammal', 'dinotherium', 'bird', 'conservationist', 'gallery forest', 'zoographer', 'wetland', 'zoophagy', 'predatory animal', 'bird table', 'sea cow', 'anhima', 'tropical rain forest', 'sea elephant', 'sea leopard', 'aquatic bird', 'birdwatch', 'zoopathology', 'wild dog', 'marine animal', 'african elephant', 'sanctuary', 'indian elephant', 'woodland', 'elephant bird', 'birdling', 'birdcatching', 'prairie wolf', 'rangership', 'snail darter', 'hairbird', 'zoological', 'wildgrave', 'kangaroo rat', 'exotic', 'indigenous', 'aquatic', 'terrestrial', 'endangered', 'migratory', 'coastal', 'captive', 'vertebrate', 'arctic', 'threatened', 'fascinating', 'nocturnal', 'mammalian', 'scenic', 'endemic', 'ocean', 'underwater', 'winged', 'amazonian', 'predatory', 'fish', 'plants', 'forests', 'forestry', 'soil', 'timber', 'cattle', 'insects', 'soils','wildlands','wildflowers', 'pesticides', 'fishing', 'extinction', 'woods', 'sanctuaries', 'ecosystem', 'elephants', 'mammals', 'outdoors', 'ducks', 'poaching', 'harvesting', 'watersheds', 'decimation', 'waterfalls', 'forage', 'invertebrates', 'feeding', 'extermination', 'pastoralists', 'shellfish', 'whales', 'buffalo', 'songbirds', 'eagles', 'seals', 'squirrels','hunted', 'endanger','depleted', 'inhabit', 'exterminated','forest ranger','game warden', 'gamekeeper', 'park','natural resources', 'panda', 'preservationist', 'preserved', 'safari', 'serengeti', 'agricultural','biosphere', 'environmental', 'freshwater', 'geriatric', 'herbivores', 'wildland']


Notice that there are some repeated words in this list. Let's extract the unique words.

In [23]:
unique_nature_words = set(selected_nature_words)
len(unique_nature_words)

211

We have `211` unique nature related words.

In [27]:
a = ["africa", "subsaharan africa", "african", "continent", "sub-saharan africa"]

len(a)

5

~ End of notebook ~